### PR TITLE
What's New (1/2)

### PR DESCRIPTION
**Pull request by barmac**
Related to https://github.com/camunda/camunda-modeler/issues/2303

This includes the visual part of the "What's New" overlay. Usage tracking will be included in a second PR.

### CODE REVIEW

**Comment by andreasgeier** [comment]
Please update the style so that we have a spacing of 12px on all edges and above and below the divider line. It should look like:

Screen Shot 2021-06-14 at 17 45 09

**Comment by andreasgeier** [comment]
@barmac Can you provide an artifact where I can check the text-overflow? 

We also need to ensure the styling for a structured release note with headline, multiple paragraphs, bullet points. I would do that with the overflow artifact as well.

Example:

Screen Shot 2021-06-14 at 17 56 48

**Comment by andreasgeier** [comment]
Please adjust the static link targets:

- [Release Notes on Camunda blog](https://camunda.com/blog/category/modeling/?tag=camunda-modeler&release-note)
- [Camunda Modeler docs](https://docs.camunda.org/manual/latest/modeler/)
- [Changelog on GitHub](https://github.com/camunda/camunda-modeler/blob/master/CHANGELOG.md)

They are general links and do not point to a specific version! Specific deep links might be included in the release note.

**Comment by andreasgeier** [comment]
I assume this PR does not include the menu entry (can't find it). Is there another PR for it?

**Comment by andreasgeier** [comment]
How can I test the unread marker? It was never visible on my artifact.

**Comment by barmac** [comment]
@andreasgeier I've triggered another build with changes applied. The unread marker should be visible when you open the app for the first time with the new version.

**Comment by barmac** [comment]
With updates:

image

image

image

**Comment by barmac** [comment]
More content:

image

image

image

**Review by andreasgeier — APPROVED** [review]
✅ &nbsp; LGTM

Great work, thx @barmac! Please do not forget to add a piece of information to the release workflow of how an update notice can be structured (which tags are supported). 
I would also recommend a visual test case before we release new versions (how they appear in the update check and the what's new overlay).

**Comment by nikku** [comment]
@andreasgeier See also https://github.com/camunda/camunda-modeler/issues/2303 for the things that @barmac does not plan to forget :wink:.

**Comment by andreasgeier** [comment]
Found a functional problem in the artifact: shouldn't the version information for this artifact be `4.8.1-2303-version-number`?

File: client/webpack.config.js [hunk 1]
```diff
@@ -106,7 +106,8 @@ module.exports = {
       'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
       'process.env.SENTRY_DSN': JSON.stringify(SENTRY_DSN),
       'process.env.ET_ENDPOINT': JSON.stringify(ET_ENDPOINT),
-      'process.env.UPDATES_SERVER_PRODUCT_NAME': JSON.stringify(UPDATES_SERVER_PRODUCT_NAME)
+      'process.env.UPDATES_SERVER_PRODUCT_NAME': JSON.stringify(UPDATES_SERVER_PRODUCT_NAME),
+      'process.env.VERSION': JSON.stringify(pkg.version)
```
Comment (nikku) [inline client/webpack.config.js:110]: Should we [get rid of passing the version around dynamically](https://github.com/camunda/camunda-modeler/blob/develop/app/lib/index.js#L82) entirely?

Feels like a code smell to me to access a thing (the version) in two different ways.

**Review by nikku — COMMENTED** [review]
All nicely encapsulated. Very good :clap:

**Comment by barmac** [comment]
> Found a functional problem in the artifact: shouldn't the version information for this artifact be `4.8.1-2303-version-number`?

As discussed, as part of this PR, I will also:
* [ ] remove version from the title bar on the nightly
* [ ] use version from nightly/build artifacts in the version info button

File: client/webpack.config.js (hunk 1 above)
Comment (barmac) [reply client/webpack.config.js:110]: I think I will just use [Metadata](https://github.com/camunda/camunda-modeler/blob/develop/client/src/util/Metadata.js#L25) instead. Let me adjust the PR.

**Comment by barmac** [comment]
image

Done.

File: client/src/plugins/version-info/VersionInfo.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+const CONFIG_KEY = 'versionInfo';
+
+export class VersionInfo extends PureComponent {
+
+  constructor(props) {
+    super(props);
+
+    this._version = Metadata.version;
+    if (process.env.NODE_ENV === 'test') {
```
Comment (nikku) [inline client/src/plugins/version-info/VersionInfo.js:33]: Why don't we rely on the version provided by Metadata.version during TEST, too?

**Review by nikku — CHANGES_REQUESTED** [review]

File: client/src/plugins/version-info/VersionInfo.js (hunk 2 above)
Comment (barmac) [reply client/src/plugins/version-info/VersionInfo.js:33]: No reason. I'll remove that.

**Review by nikku — APPROVED** [review]
:clap:

**Pull request merged**